tests/test_parse.py

```diff
@@ -9,7 +9,8 @@
 
 
 def test_get_all_courses():
-    all_courses = get_all_courses()
+    scraper = ScrapeMateCourses()
+    all_courses = scraper.get_all_courses()
```

Comment:
when you have implemented tests, it isn't recommended to update them because there is always a chance that you'll break the test + your logic is incorrect = app works incorrectly